apps/web/src/components/transactions/HexEncodedData/styles.module.css

```diff
@@ -7,4 +7,5 @@
 .encodedData button {
   padding-top: 0;
   padding-bottom: 0;
+  padding-left: 0;
```

Comment:
Left-aligns the data.

<img width="285" alt="Screenshot 2025-03-21 at 07 23 57" src="https://github.com/user-attachments/assets/83a58ebb-9d17-4c02-8b11-ce0d9cb93f40" />